evaluation/assert_check_format.py

```diff
@@ -0,0 +1,183 @@
+import json
+import pprint
+from typing import Any
+
+
+def compare_format(
+    generated_output: list[dict[str, Any]], expected_output: dict[str, Any]
+) -> tuple[bool, str]:
+    # fail if the generated output isn't a list of dicts
+    if not isinstance(generated_output, list):
+        return False, "output is not list"
+
+    # loop through each dictionary and check the keys match
+    for claim in generated_output:
+        if not isinstance(claim, dict):
+            return False, "claims are not all dicts"
+
+        if claim.keys() != expected_output.keys():
+            return (
+                False,
+                f"output claim does not have the expected keys: found {list(claim.keys())}, expected {list(expected_output.keys())}",
+            )
+
+        if "labels" not in claim or not isinstance(claim["labels"], dict):
+            return False, "labels are not present"
+
+        if claim["labels"].keys() != expected_output["labels"].keys():
+            return (
+                False,
+                f"the labels are incorrect: found {list(claim['labels'].keys())}, expected {list(expected_output['labels'].keys())}",
+            )
+
+    return True, "output is correctly formatted"
+
+
+def get_assert(output: str, context: dict[str, Any]) -> bool | float | dict[str, Any]:
+    variables = context["vars"]
+    generated_output = json.loads(output)
+    expected_output = json.loads(variables["expected_output"])
+
+    # fail if the output claims don't match the expected claims formatting
+    formats_match, reason = compare_format(generated_output, expected_output[0])
```

Comment:
Why  `expected_output[0]` - what about the other items in the  `expected_output` list?

Reply:
In the expected output, every claim should be the same format. It's just checking field names and stuff, so unless we're deliberately giving it bad evaluation data, the first should always be enough